templates/app/webpack-default/template/src/profile.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * This example demonstrates how a simple profile view can be created with
+ * Titanium. The profile includes
+ *
+ * - Profile header
+ * - A styled TableView with items that open new windows
+ * - ScrollView to allow scrolling of the whole view on smaller devices
+ */
+
+/* global tabGroup */
+
+import { createWindow, createTab, faIcon } from './utils';
+
+const title = 'Profile';
+const window = createWindow({ title, layout: 'vertical' });
+
+const scrollView = Ti.UI.createScrollView({ layout: 'vertical' });
```

Comment:
You need to set creation property `scrollType` to `"vertical"` or else a warning will be logged on Android. This is because Android's native `ScrollView` does not support scrolling both vertically and horizontally like iOS. It needs to be told upon creation which direction to support. (It defaults to vertical, but will log a warning if the property is missing.)
```javascript
const scrollView = Ti.UI.createScrollView({
	layout: 'vertical',
	scrollType: 'vertical'
});
```

Reply:
Would it make sense to have a default value of `vertical` for that? Most of the time when i use scroll views i want them to be vertical anyway.

Reply:
It already defaults to "vertical" now, but it logs a warning stating that you should explicitly set it. This warning comes from our code and it's been this way for years.